src/server/GameState.java

```diff
@@ -86,11 +86,17 @@ private void generateEnemies() {
     }
 
     private void generateShields() {
+        ShieldFragmentServerEntity shieldEntity = addShieldFragmentEntity(1,1);
         for (int i = 100; i < 200; i += 10) {
             for (int j = 500; j < 550; j += 10) {
-                addShieldFragmentEntity(i, j);
+                ShieldFragmentServerEntity shieldCopy = shieldEntity.deepCopy();
+                shieldCopy.setX(i);
+                shieldCopy.setY(j);
+                shieldFragmentEntities.put(shieldCopy.getId(), shieldCopy);
+                notifyObservers(new EntityUpdateEvent(shieldCopy, false));
             }
         }
+        removeEntity(shieldEntity.getId(), EntityType.SHIELD);
```

Comment:
Maybe create shieldEntity at 100, 500 coordinates and increment the start of the for loops, so we wouldn't have to remove it later